scripts/schemas/twister/testcase-schema.yaml

```diff
@@ -62,6 +62,12 @@ mapping:
           "repeat":
             type: int
             required: no
+          "pytest_root":
+            type: str
+            required: no
+          "pytest_timeout":
```

Comment:
no need for a new timeout, use the existing one

Reply:
fixed